lib/logthrdestdrv.c

```diff
@@ -243,6 +274,7 @@ _process_result(LogThreadedDestDriver *self, gint result)
       break;
 
     case WORKER_INSERT_RESULT_QUEUED:
+      self->enable_flush_timeout = TRUE;
```

Comment:
Should we set this variable to false in _accept_batch and _drop_batch?

Reply:
Oh ok I thought we could use self->flush_timeout, but that would not work because it can be set with global options, so we need an additional filtering mechanism not to start the timer for drivers that do not support it. Marking as resolved.